src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -236,6 +239,17 @@ object PlayReleaseBase extends AutoPlugin {
 
 }
 
+object PlayLicensePlugin extends AutoPlugin {
```

Comment:
I'd just call it `PlayWhitesourcePlugin` since whitesource does more than just license checking.